install.md

```diff
@@ -0,0 +1,100 @@
+
+## Pre-requisites
+
+You must have a the ability to run root containers in a cluster
+with at least 12 vCPUs and 45G of memory and a minumum of three nodes.
+You will also need to provide a docker compatable registry. 
+
+## Configure your Registry
+In order to make this install simple to walk through we recomend you 
+store your docker registry details in a an environment variable. This
+install guide uses gcr on gke. 
+
+```
+export KF_REGISTRY=gcr.io/<PROJECT_ID>
+```
+
+
+## Install Istio && Knative
```

Comment:
This looks like the install instructions found here: https://github.com/knative/docs/blob/master/docs/install/Knative-with-GKE.md

Can we just link to that? 

Or better yet, just link to https://github.com/knative/docs/blob/master/docs/install/ so the operator can install knative on whatever version of K8s they have.

Reply:
This gets at the discussion happening over in the google group a bit. It's mostly the same, but it does give us some ability to swap out components and keep users on our workflow towards `kf push`. I think its worth the minimal duplication here.

Reply:
My only worry is they will be a hassle to keep up to date. For example, you are referencing version 5. That is for sure going to change over time.

Reply:
This way we pin to a known version tho at least.